dotnet/src/Connectors/Connectors.Ollama.UnitTests/Client/OllamaStreamJsonParserTest.cs

```diff
@@ -0,0 +1,185 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.SemanticKernel.Connectors.Ollama.Core;
+using Xunit;
+
+namespace SemanticKernel.Connectors.Ollama.UnitTests;
+
+public class OllamaStreamJsonParserTest
```

Comment:
Instead of using sync version of stream parser, copy StreamJsonParser (which is async) and related unit tests from gemini branch

Reply:
This can be in a next PR. This is UT focused for now.

@BLaZeKiLL just drop the JsonParser tests as they will be done by the SemanticKernel.UnitTests.